api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentController.java

```diff
@@ -64,4 +68,24 @@ public PaymentController(PaymentService<PaymentFeeLink, String> paymentService)
 
         return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
+
+    @ApiOperation(value = "Get payments for between dates", notes = "Get payments for between dates, enter the date in format dd-MM-yyyy")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Payments retrieved"),
+        @ApiResponse(code = 400, message = "Bad request"),
+        @ApiResponse(code = 404, message = "Payments not found")
+    })
+    @RequestMapping(value = "/payments", method = GET)
+    public ResponseEntity<?> retrievePayments(@RequestParam(name = "start_date", required = false) String startDate,
+                                              @RequestParam(name = "end_date", required = false) String endDate,
+                                              @RequestParam(name = "payment_method")PaymentMethodUtil paymentMethodType) {
```

Comment:
Map to string otherwise you get Enum ValueOf exception for invalid values.

Reply:
Added a validation message now for incorrect/invalid method type